src/main/app-menu.ts

```diff
@@ -494,6 +495,7 @@ export function setupMenu (
       label: "Start",
       enabled: machineWaits,
       click: async () => {
+        mainStore.dispatch(setRestartTarget("machine"));
```

Comment:
This hinting looks a bit messy on the first glance. This is because they're scattered across the App. But at the same time such an approach takes the need in meddling with commands/machine controller internals off us.